docs/get_started/raspbian_setup.md

```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
     git clone https://github.com/dmlc/mxnet.git --recursive
 ```
 
-Then build the full MxNet library, ensuring the ARM flag is enabled (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the compiled library. You can do this by compiling with the following code:
 ```bash
     cd mxnet
     export ARM=1
+    export USE_OPENCV = 0
     make
 ```
 
-If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the loaded library. You can do this by compiling with the following code:
+Otherwise you can build the full MxNet library: 
 ```bash
     cd mxnet
     export ARM=1
-    export USE_OPENCV = 0
     make
 ```
 
+Whenever you are compiling MXNet for the Pi you need to ensure the "ARM" environment variable is set to 1 (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
```

Comment:
For ARM compatibility, set the ARM environment variable to 1. This removes the x86-specific -msse flag from the NNVM submodule's makefile.